src/theme/material/variants/dark.m.css

```diff
@@ -1,4 +1,6 @@
-.root {
+.root,
+.root *::before,
```

Comment:
@tomdye what's this about?

Reply:
This is necessary to pass the css variables down to the before and after selectors

Reply:
Why do the before / after nodes need them?